File: python/exercise06/retirement_calculator.py
from datetime import datetime

age = int(input("What is your current age? "))
retire_age = int(input("At what age would you like to retire? "))

# quit program if user is already retire
if age >= retire_age:
	print("You are already retired!!")
	exit(0)

current_year = datetime.now().year

print(f"You have {retire_age - age} years left until you can retire.")
print(f"It's {current_year}, so you can retire in {current_year + (retire_age - age)}.")

File: javascript/exercise06/retirement-calculator.js
import { input } from "pynput";

const age = Number(await input("What is your current age? "));
const retireAge = Number(await input("At what age would you like to retire? "));

// quit program if user is already retire
if (age >= retireAge) {
	console.log("You are already retired!!");
	process.exit();
}

const currentYear = new Date().getFullYear();

console.log(`You have ${retireAge - age} years left until you can retire.`);
console.log(
	`It's ${currentYear}, so you can retire in ${currentYear + (retireAge - age)}.`,
);
